src/sagemaker/workflow/condition_step.py

```diff
@@ -95,7 +95,7 @@ def properties(self):
 
 
 @attr.s
-class JsonGet(Expression):
+class JsonGet(Expression):  # pragma: no cover
```

Comment:
this make senses, as we deprecated the JsonGet defined in `condition_step.py`

Reply:
Yes. This PR also removed the usage of such JsonGet defined in `condition_step.py`